src/main/java/millom/sandbox/mars/weather/service/JdkService.java

```diff
@@ -0,0 +1,34 @@
+package millom.sandbox.mars.weather.service;
+
+import jakarta.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import millom.sandbox.mars.weather.CustomException.JdkException;
+import millom.sandbox.mars.weather.mapper.JdkMapper;
+import millom.sandbox.mars.weather.record.ReleaseNames;
+
+public class JdkService {
+
+  @Inject
+  private JdkClintService jdkClintService;
+  @Inject
+  private JdkMapper jdkMapper;
+
+  public List<String> getReleaseNamesMapping(String architecture) throws JdkException {
+    List<String> allPagesReleaseNames = new ArrayList<>();
+    String marsWeatherAsString = "";
```

Comment:
Better to change this variable name to something relevant. F.ex. to 'pageContent' or something.

Reply:
`public List<String> getReleaseNamesMapping(String architecture) throws JdkException {
    List<String> allPagesReleaseNames = new ArrayList<>();
    String pageContent= "";`